### DISCUSSION TITLE
获取pag帧数据时，保存为png图片发现文字没了

### DISCUSSION THREAD

**Discussion by jackTJT** [Tencent/libpag#1963] [General] [answered]
在linux上进行实验时，用ae做了一个pag图片，将帧数据保存下来时发现文字没了，用pagViewer预览是正常的，使用 pagFile->numTexts()也打印出有两个可以显示
- pagViewer预览情况：
![GHYX0T8BS_2DZX64Z~S96](https://github.com/Tencent/libpag/assets/96832322/0b4c727d-88d9-4e5e-8209-c615d30122be)
- 保存下来的帧数据显示情况
![2$OV4QVQ306EU4K7VIPS](https://github.com/Tencent/libpag/assets/96832322/86d41013-c30d-424f-99f9-400d3e0b246c)
-  pagFile->numTexts()打印
![6L2XLW `XA~7SN9Z5K8](https://github.com/Tencent/libpag/assets/96832322/3f221473-5af3-4d02-b148-a18f11c288b1)

**Comment by kevingpqi123** [comment, marked as answer]
具体看下这个提问：https://pag.art/docs/faq.html#%E9%97%AEpag-%E4%B8%AD%E5%AD%97%E4%BD%93%E5%8A%A0%E8%BD%BD%E7%9A%84%E7%AD%96%E7%95%A5%E6%98%AF%E4%BB%80%E4%B9%88-linux-%E7%AB%AF%E6%96%87%E6%9C%AC%E4%B8%BA%E5%95%A5%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA
如果不了解怎么操作，看下源码 linux 目录下的 main.cpp
